AUTO_CURSEFORGE failing to update - new modloader type

### Describe the problem

When trying to update modpack https://www.curseforge.com/minecraft/modpacks/statech-industry from v1.1.2 to v.1.1.3, container fails with the following error:

I suspect one of the included (unsure which) mods has a "latestFileIndex" with a modloader outside the normal range.

### Container definition

```{
    "AppArmorProfile": "docker-default",
    "Args": [],
    "Config": {
        "AttachStderr": false,
        "AttachStdin": false,
        "AttachStdout": false,
        "Cmd": [],
        "Domainname": "",
        "Entrypoint": [
            "/start"
        ],
        "Env": [
            "EULA=TRUE",
            "TYPE=AUTO_CURSEFORGE",
            "CF_PAGE_URL=https://www.curseforge.com/minecraft/modpacks/statech-industry",
            "MEMORY=8G",
            "PVP=false",
            "LANG=en_US.UTF-8",
            "LANGUAGE=en_US:en",
            "LC_ALL=en_US.UTF-8",
            "UID=1000",
            "GID=1000",
            "VERSION=LATEST",
            "ENABLE_RCON=true",
            "RCON_PORT=25575",
            "RCON_PASSWORD=PBQ8mPJHNUyTbCSFUhEt",
            "ENABLE_AUTOPAUSE=false",
            "AUTOPAUSE_TIMEOUT_EST=3600",
            "AUTOPAUSE_TIMEOUT_KN=120",
            "AUTOPAUSE_TIMEOUT_INIT=600",
            "AUTOPAUSE_PERIOD=10",
            "AUTOPAUSE_KNOCK_INTERFACE=eth0",
            "OPS=AerikAwesome",
            "DIFFICULTY=normal",
            "ALLOW_FLIGHT=TRUE",
            "ENABLE_AUTOSTOP=false",
            "AUTOSTOP_TIMEOUT_EST=3600",
            "AUTOSTOP_TIMEOUT_INIT=1800",
            "AUTOSTOP_PERIOD=10",
            "JAVA_HOME=/opt/java/openjdk",
            "JAVA_VERSION=jdk-17.0.5+8",
            "PATH=/opt/java/openjdk/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
            "DEBUG=false",
            "CF_API_KEY=redacted",
            "USE_AIKAR_FLAGS=true",
            "CF_EXCLUDE_MODS=627566 704346 356643 475358 406343 393563 407206 398502 619718 365727 337243 547694 401978 541329 566140 417510 280294 313219 365615 629373 271492 459496 495267 455508 679177 325492 356461 308702 365521 363104 511319 385463 447673 394468 282313 521126 521594 547358"
        ],
        "ExposedPorts": {
            "25565/tcp": {},
            "25565/udp": {},
            "25575/tcp": {},
            "8080/tcp": {}
        },
        "Healthcheck": {
            "Interval": 5000000000,
            "Retries": 24,
            "StartPeriod": 60000000000,
            "Test": [
                "CMD-SHELL",
                "mc-health"
            ]
        },
        "Hostname": "fb464a135cb7",
        "Image": "itzg/minecraft-server:latest",
        "Labels": {
            "org.opencontainers.image.authors": "Geoff Bourne <itzgeoff@gmail.com>",
            "org.opencontainers.image.created": "2021-12-20T13:09:50.887Z",
            "org.opencontainers.image.description": "Docker image that provides a Minecraft Server that will automatically download selected version at startup",
            "org.opencontainers.image.licenses": "Apache-2.0",
            "org.opencontainers.image.ref.name": "ubuntu",
            "org.opencontainers.image.revision": "c15e231d7e8bec6c909d74904e61e8deae65e7ba",
            "org.opencontainers.image.source": "https://github.com/itzg/docker-minecraft-server",
            "org.opencontainers.image.title": "docker-minecraft-server",
            "org.opencontainers.image.url": "https://github.com/itzg/docker-minecraft-server",
            "org.opencontainers.image.version": "master"
        },
        "OnBuild": null,
        "OpenStdin": false,
        "StdinOnce": false,
        "StopSignal": "SIGTERM",
        "Tty": false,
        "User": "",
        "Volumes": {
            "/data": {}
        },
        "WorkingDir": "/data"
    },
    "Created": "2023-07-30T09:46:39.766043436Z",
    "Driver": "overlay2",
    "ExecIDs": null,
    "GraphDriver": {
        "Data": {
            "LowerDir": "/sdata/docker/overlay2/5b80a693d58f5890fb040d919cff64c32cf5f99e0c520b3720f06a20bec28360-init/diff:/sdata/docker/overlay2/dbe93d16f8c83fb4388cd58f38bcebe154cb35c74a3be2882d3fa181b5d19eea/diff:/sdata/docker/overlay2/58f4e061f9466c16960c7378ced93d0dff0050387da9d4bf90201ec51f10d8c4/diff:/sdata/docker/overlay2/9afb1fbe6bb6cd8b7ddd3b99a08f50244e8979c8f1d7708b57d73d1035f58255/diff:/sdata/docker/overlay2/2a2ab1e03ed77aa4fc657123db0ffc450f09add07197fb1011909e1297a8c628/diff:/sdata/docker/overlay2/ed3ca0de79b91d17ea6ffa14e735d6c5350d2ca8fa0b60b2802c695fcb308fd0/diff:/sdata/docker/overlay2/6a7c1259113d58cfa9d4bc3f8323ab3238d2161e2f868fcf75a0f0182bba4978/diff:/sdata/docker/overlay2/d1863be4626248257ea23b099c983511b80947f22bb8180df868c8d63637c83e/diff:/sdata/docker/overlay2/e158346c70b3241ab43ccf002e11ebea1d3d3f3139097cbcf635bd736e3a2fde/diff:/sdata/docker/overlay2/0746429f54b8425d724a1838f92f3780b44759e5a0a8c35c34d2224316a19b70/diff:/sdata/docker/overlay2/c8183f5de0dd1f6f0be7a952f208eebfe68fd394775b593e22f7472df2845e23/diff:/sdata/docker/overlay2/ece9b9c59f500ff4c12f3fd7bb7d0721f593fe9212fdfbc6c40711aa5ffc3838/diff:/sdata/docker/overlay2/a315f8007201c3a091fcc1788677920d49018f6e0dbae1b305f4517a5254c957/diff:/sdata/docker/overlay2/7f5ed484bc6bccf4cd7dd63ba89a64d689c3eda829d06439d16cb3046450746c/diff:/sdata/docker/overlay2/665097ff50f80525e584c237f65757782043cacc3e4a457612e12e0da40c0602/diff:/sdata/docker/overlay2/2474e4469b7ad605ab7a6c7d3c13a5c24143b4562967a2a4b968df6beda9f312/diff:/sdata/docker/overlay2/e51b4e40cfa194230b298b58e625f893b7f39b78fd81209d0857830c36fbae02/diff:/sdata/docker/overlay2/2ece1024989b0d4024a8804ba096ce96dadb4d48f2d3e0db12bb5ccb921066f4/diff:/sdata/docker/overlay2/494590b96c175f373fba0fe403e0cf09ebea3428a6052ac50cecadcb7876336c/diff:/sdata/docker/overlay2/ebc6a3c3db4548dfaad76f543120ca060ee4553588a24534bef5dcbe8dd310ec/diff:/sdata/docker/overlay2/b75da5f7fe32a41a613553cc76d9de1ce72a7e55f79d1c6c73190998d55ed96f/diff:/sdata/docker/overlay2/2e457e4f552e125668bd2faab2961e2e3ba738f0d5c2ac628691cb23876d35ff/diff:/sdata/docker/overlay2/823f358816a18da93cd2539225e66c84e66746f74e994552446baae6b4563bb6/diff:/sdata/docker/overlay2/f035f329d3c429efde687c32e57fd9615c4bfa3fdd7e6042d6c72cb18434d283/diff",
            "MergedDir": "/sdata/docker/overlay2/5b80a693d58f5890fb040d919cff64c32cf5f99e0c520b3720f06a20bec28360/merged",
            "UpperDir": "/sdata/docker/overlay2/5b80a693d58f5890fb040d919cff64c32cf5f99e0c520b3720f06a20bec28360/diff",
            "WorkDir": "/sdata/docker/overlay2/5b80a693d58f5890fb040d919cff64c32cf5f99e0c520b3720f06a20bec28360/work"
        },
        "Name": "overlay2"
    },
    "HostConfig": {
        "AutoRemove": false,
        "Binds": [
            "/sdata/mcStatech:/data"
        ],
        "BlkioDeviceReadBps": null,
        "BlkioDeviceReadIOps": null,
        "BlkioDeviceWriteBps": null,
        "BlkioDeviceWriteIOps": null,
        "BlkioWeight": 0,
        "BlkioWeightDevice": null,
        "CapAdd": [
            "AUDIT_WRITE",
            "CHOWN",
            "DAC_OVERRIDE",
            "FOWNER",
            "FSETID",
            "KILL",
            "MKNOD",
            "NET_BIND_SERVICE",
            "NET_RAW",
            "SETFCAP",
            "SETGID",
            "SETPCAP",
            "SETUID",
            "SYS_CHROOT"
        ],
        "CapDrop": [
            "AUDIT_CONTROL",
            "BLOCK_SUSPEND",
            "DAC_READ_SEARCH",
            "IPC_LOCK",
            "IPC_OWNER",
            "LEASE",
            "LINUX_IMMUTABLE",
            "MAC_ADMIN",
            "MAC_OVERRIDE",
            "NET_ADMIN",
            "NET_BROADCAST",
            "SYSLOG",
            "SYS_ADMIN",
            "SYS_BOOT",
            "SYS_MODULE",
            "SYS_NICE",
            "SYS_PACCT",
            "SYS_PTRACE",
            "SYS_RAWIO",
            "SYS_RESOURCE",
            "SYS_TIME",
            "SYS_TTY_CONFIG",
            "WAKE_ALARM"
        ],
        "Cgroup": "",
        "CgroupParent": "",
        "CgroupnsMode": "host",
        "ConsoleSize": [
            0,
            0
        ],
        "ContainerIDFile": "",
        "CpuCount": 0,
        "CpuPercent": 0,
        "CpuPeriod": 0,
        "CpuQuota": 0,
        "CpuRealtimePeriod": 0,
        "CpuRealtimeRuntime": 0,
        "CpuShares": 0,
        "CpusetCpus": "",
        "CpusetMems": "",
        "DeviceCgroupRules": null,
        "DeviceRequests": null,
        "Devices": [],
        "Dns": [],
        "DnsOptions": [],
        "DnsSearch": [],
        "ExtraHosts": [],
        "GroupAdd": null,
        "IOMaximumBandwidth": 0,
        "IOMaximumIOps": 0,
        "IpcMode": "private",
        "Isolation": "",
        "KernelMemory": 0,
        "KernelMemoryTCP": 0,
        "Links": null,
        "LogConfig": {
            "Config": {},
            "Type": "json-file"
        },
        "MaskedPaths": [
            "/proc/asound",
            "/proc/acpi",
            "/proc/kcore",
            "/proc/keys",
            "/proc/latency_stats",
            "/proc/timer_list",
            "/proc/timer_stats",
            "/proc/sched_debug",
            "/proc/scsi",
            "/sys/firmware"
        ],
        "Memory": 0,
        "MemoryReservation": 0,
        "MemorySwap": 0,
        "MemorySwappiness": null,
        "NanoCpus": 0,
        "NetworkMode": "minecraft-net",
        "OomKillDisable": null,
        "OomScoreAdj": 0,
        "PidMode": "",
        "PidsLimit": null,
        "PortBindings": {
            "25565/tcp": [
                {
                    "HostIp": "",
                    "HostPort": "25570"
                }
            ],
            "25565/udp": [
                {
                    "HostIp": "",
                    "HostPort": "25570"
                }
            ],
            "25575/tcp": [
                {
                    "HostIp": "",
                    "HostPort": "25580"
                }
            ],
            "8080/tcp": [
                {
                    "HostIp": "",
                    "HostPort": "25560"
                }
            ]
        },
        "Privileged": false,
        "PublishAllPorts": false,
        "ReadonlyPaths": [
            "/proc/bus",
            "/proc/fs",
            "/proc/irq",
            "/proc/sys",
            "/proc/sysrq-trigger"
        ],
        "ReadonlyRootfs": false,
        "RestartPolicy": {
            "MaximumRetryCount": 0,
            "Name": "unless-stopped"
        },
        "Runtime": "runc",
        "SecurityOpt": null,
        "ShmSize": 67108864,
        "UTSMode": "",
        "Ulimits": null,
        "UsernsMode": "",
        "VolumeDriver": "",
        "VolumesFrom": null
    },
    "HostnamePath": "/sdata/docker/containers/8e059f8ede543fac924254be2917fabca7050fea320ff9fa250d62facb16ffee/hostname",
    "HostsPath": "/sdata/docker/containers/8e059f8ede543fac924254be2917fabca7050fea320ff9fa250d62facb16ffee/hosts",
    "Id": "8e059f8ede543fac924254be2917fabca7050fea320ff9fa250d62facb16ffee",
    "Image": "sha256:9e379191f5f89860125fb739f24535be974a6ff9ca03f0a10327b6223547968b",
    "LogPath": "/sdata/docker/containers/8e059f8ede543fac924254be2917fabca7050fea320ff9fa250d62facb16ffee/8e059f8ede543fac924254be2917fabca7050fea320ff9fa250d62facb16ffee-json.log",
    "MountLabel": "",
    "Mounts": [
        {
            "Destination": "/data",
            "Mode": "",
            "Propagation": "rprivate",
            "RW": true,
            "Source": "/sdata/mcStatech",
            "Type": "bind"
        }
    ],
    "Name": "/mcStatech",
    "NetworkSettings": {
        "Bridge": "",
        "EndpointID": "",
        "Gateway": "",
        "GlobalIPv6Address": "",
        "GlobalIPv6PrefixLen": 0,
        "HairpinMode": false,
        "IPAddress": "",
        "IPPrefixLen": 0,
        "IPv6Gateway": "",
        "LinkLocalIPv6Address": "",
        "LinkLocalIPv6PrefixLen": 0,
        "MacAddress": "",
        "Networks": {
            "minecraft-net": {
                "Aliases": [
                    "8e059f8ede54",
                    "fb464a135cb7"
                ],
                "DriverOpts": null,
                "EndpointID": "",
                "Gateway": "",
                "GlobalIPv6Address": "",
                "GlobalIPv6PrefixLen": 0,
                "IPAMConfig": {},
                "IPAddress": "",
                "IPPrefixLen": 0,
                "IPv6Gateway": "",
                "Links": null,
                "MacAddress": "",
                "NetworkID": "bda513fdeeef77f065015814b4a80cdeaf826a454ed94df651efb78e994d7ca5"
            }
        },
        "Ports": {},
        "SandboxID": "82b7fb6d10018ecc439a2f5b2bc64165c41ef907219e3b675f74ba4b22c5d960",
        "SandboxKey": "/var/run/docker/netns/82b7fb6d1001",
        "SecondaryIPAddresses": null,
        "SecondaryIPv6Addresses": null
    },
    "Path": "/start",
    "Platform": "linux",
    "ProcessLabel": "",
    "ResolvConfPath": "/sdata/docker/containers/8e059f8ede543fac924254be2917fabca7050fea320ff9fa250d62facb16ffee/resolv.conf",
    "RestartCount": 60,
    "State": {
        "Dead": false,
        "Error": "",
        "ExitCode": 1,
        "FinishedAt": "2023-07-30T10:40:31.877776509Z",
        "Health": {
            "FailingStreak": 0,
            "Log": [],
            "Status": "unhealthy"
        },
        "OOMKilled": false,
        "Paused": false,
        "Pid": 0,
        "Restarting": true,
        "Running": true,
        "StartedAt": "2023-07-30T10:40:29.311100229Z",
        "Status": "restarting"
    }
}
```


### Container logs

```[init] Running as uid=1000 gid=1000 with /data as 'drwxrwxr-x 23 1000 1000 4096 Jul 26 16:30 /data'
[mc-image-helper] 09:46:42.265 INFO  : Processing modpack 'StaTech 1.1.3' (statech-industry) @ 863689:4670282
[mc-image-helper] 09:46:42.916 INFO  : Excluding mod file 'Light Overlay (Rift/Forge/Fabric)' (light-overlay) due to configuration
[mc-image-helper] 09:46:43.033 INFO  : Mod file mods/lithium-fabric-mc1.19.2-0.11.1.jar already exists
[mc-image-helper] 09:46:43.034 INFO  : Mod file mods/carryon-fabric-1.19.2-2.1.0.20.jar already exists
[mc-image-helper] 09:46:43.034 INFO  : Mod file mods/item-filters-fabric-1902.2.9-build.51.jar already exists
[mc-image-helper] 09:46:43.035 INFO  : Mod file mods/polaroidcamera-1.4.1.jar already exists
[mc-image-helper] 09:46:43.036 INFO  : Mod file mods/sliceanddice-fabric-2.2.1.jar already exists
[mc-image-helper] 09:46:43.038 INFO  : Mod file mods/scannable-MC1.19.2-fabric-1.7.7+dc5ea09.jar already exists
[mc-image-helper] 09:46:43.053 INFO  : Excluding mod file 'Adaptive Tooltips' (adaptive-tooltips) due to configuration
[mc-image-helper] 09:46:43.095 INFO  : Mod file mods/findme-3.1.0-fabric.jar already exists
[mc-image-helper] 09:46:43.096 INFO  : Mod file mods/wthit-fabric-5.17.0.jar already exists
[mc-image-helper] 09:46:43.104 INFO  : Mod file mods/spiceoffabric-1.6.0-beta.4+1.19.2.jar already exists
[mc-image-helper] 09:46:43.128 INFO  : Mod file mods/alternate-current-mc1.19-1.7.0.jar already exists
[mc-image-helper] 09:46:43.162 INFO  : Mod file mods/snow-under-trees-fabric-1.0.0+1.19.2.jar already exists
[mc-image-helper] 09:46:43.166 INFO  : Mod file mods/YungsBetterWitchHuts-1.19.2-Fabric-2.1.0.jar already exists
[mc-image-helper] 09:46:43.166 INFO  : Mod file mods/VisualWorkbench-v4.2.4-1.19.2-Fabric.jar already exists
[mc-image-helper] 09:46:43.167 INFO  : Mod file mods/CorgiLib-fabric-1.19.2-1.0.0.34.jar already exists
[mc-image-helper] 09:46:43.168 INFO  : Mod file mods/Modern-Dynamics-0.6.1-beta.jar already exists
[mc-image-helper] 09:46:43.168 INFO  : Mod file mods/Iceberg-1.19.1-fabric-1.0.46.jar already exists
[mc-image-helper] 09:46:43.169 INFO  : Mod file mods/lumberaxe-1.0.1.jar already exists
[mc-image-helper] 09:46:43.169 INFO  : Mod file mods/peripheralworks-0.2.9.jar already exists
[mc-image-helper] 09:46:43.169 INFO  : Mod file mods/runes-fabric-0.9.8+1.19.jar already exists
[mc-image-helper] 09:46:43.170 INFO  : Mod file mods/Decorative Blocks-fabric-1.19.2-3.0.0.jar already exists
[mc-image-helper] 09:46:43.171 INFO  : Mod file mods/YungsBetterOceanMonuments-1.19.2-Fabric-2.1.0.jar already exists
[mc-image-helper] 09:46:43.171 INFO  : Mod file mods/antighost-1.19.1-fabric0.58.5-1.1.5.jar already exists
[mc-image-helper] 09:46:43.172 INFO  : Mod file mods/spectrum-1.6.12-1.19.2-leisure_time.jar already exists
[mc-image-helper] 09:46:43.176 INFO  : Mod file mods/forgivingvoid-fabric-1.19-7.0.0.jar already exists
[mc-image-helper] 09:46:43.303 INFO  : Mod file mods/createaddition-fabric+1.19.2-20230723a.jar already exists
[mc-image-helper] 09:46:43.313 INFO  : Mod file mods/time-in-a-bottle-1.6.2+mc1.19-fabric.jar already exists
[mc-image-helper] 09:46:43.351 INFO  : Mod file mods/betterstats-2.9.3+1.19.2.jar already exists
[mc-image-helper] 09:46:43.450 INFO  : Excluding mod file 'MemoryLeakFix' (memoryleakfix) due to configuration
[mc-image-helper] 09:46:43.487 INFO  : Mod file mods/trinkets-3.4.2.jar already exists
[mc-image-helper] 09:46:43.577 INFO  : Excluding mod file 'FPS Reducer' (fps-reducer) due to configuration
[mc-image-helper] 09:46:43.695 INFO  : Mod file mods/quickshulker-1.3.9-1.19.jar already exists
[mc-image-helper] 09:46:43.699 INFO  : Mod file mods/blockus-2.5.10+1.19.2.jar already exists
[mc-image-helper] 09:46:43.701 INFO  : Mod file mods/async-locator-fabric-1.19.2-1.3.1.jar already exists
[mc-image-helper] 09:46:43.706 INFO  : Excluding mod file 'Chat Heads' (chat-heads) due to configuration
[mc-image-helper] 09:46:43.820 INFO  : Mod file mods/spell_engine-fabric-0.9.25+1.19.jar already exists
[mc-image-helper] 09:46:43.825 INFO  : Mod file mods/kleeslabs-fabric-1.19.2-12.3.0.jar already exists
[mc-image-helper] 09:46:43.826 INFO  : Mod file mods/RoughlyEnoughItems-9.1.632.jar already exists
[mc-image-helper] 09:46:43.834 INFO  : Mod file mods/RoughlyEnoughLootTables-1.19-1.0.jar already exists
[mc-image-helper] 09:46:43.834 INFO  : Excluding mod file 'Sodium Extra' (sodium-extra) due to configuration
[mc-image-helper] 09:46:43.834 INFO  : Mod file mods/kubejs-fabric-1902.6.1-build.337.jar already exists
[mc-image-helper] 09:46:43.834 INFO  : Mod file mods/appleskin-fabric-mc1.19-2.4.1.jar already exists
[mc-image-helper] 09:46:43.834 INFO  : Mod file mods/cccbridge-mc1.19.2-fabric-v1.5.1.jar already exists
[mc-image-helper] 09:46:43.838 INFO  : Mod file mods/YungsBetterDesertTemples-1.19.2-Fabric-2.2.2.jar already exists
[mc-image-helper] 09:46:43.838 INFO  : Mod file mods/cultural-delights-fabric-0.14.10+1.19.2.jar already exists
[mc-image-helper] 09:46:43.954 INFO  : Mod file mods/geocluster-1.2.3+1.19.2.jar already exists
[mc-image-helper] 09:46:43.955 INFO  : Excluding mod file 'Sodium' (sodium) due to configuration
[mc-image-helper] 09:46:43.956 INFO  : Mod file mods/PuzzlesLib-v4.4.3-1.19.2-Fabric.jar already exists
[mc-image-helper] 09:46:43.956 INFO  : Mod file mods/resourcefullib-fabric-1.19.2-1.1.24.jar already exists
[mc-image-helper] 09:46:44.082 INFO  : Mod file mods/balm-fabric-1.19.2-4.5.7.jar already exists
[mc-image-helper] 09:46:44.083 INFO  : Mod file mods/toolleveling-fabric-1.19.2-1.4.1.jar already exists
[mc-image-helper] 09:46:44.084 INFO  : Mod file mods/Chunky-1.3.38.jar already exists
[mc-image-helper] 09:46:44.095 ERROR : 'install-curseforge' command failed. Version is 1.32.8

me.itzg.helpers.errors.GenericException: Failed to parse response body into class me.itzg.helpers.curseforge.model.GetModResponse: Cannot deserialize value of type `me.itzg.helpers.curseforge.model.ModLoaderType` from String "6": not one of the values accepted for Enum class: [0, 1, 2, 3, 4, 5]
 at [Source: (reactor.netty.ByteBufMono$ReleasingInputStream); line: 1, column: 11029] (through reference chain: me.itzg.helpers.curseforge.model.GetModResponse["data"]->me.itzg.helpers.curseforge.model.CurseForgeMod["latestFilesIndexes"]->java.util.ArrayList[1]->me.itzg.helpers.curseforge.model.FileIndex["modLoader"])
	at me.itzg.helpers.http.ObjectFetchBuilder.lambda$handleResponse$1(ObjectFetchBuilder.java:89)
	at reactor.core.publisher.MonoFlatMap$FlatMapMain.onNext(MonoFlatMap.java:132)
	at reactor.core.publisher.MonoPublishOn$PublishOnSubscriber.run(MonoPublishOn.java:181)
	at reactor.core.scheduler.SchedulerTask.call(SchedulerTask.java:68)
	at reactor.core.scheduler.SchedulerTask.call(SchedulerTask.java:28)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
	Suppressed: java.lang.Exception: #block terminated with an error
		at reactor.core.publisher.BlockingSingleSubscriber.blockingGet(BlockingSingleSubscriber.java:102)
		at reactor.core.publisher.Mono.block(Mono.java:1712)
		at me.itzg.helpers.curseforge.CurseForgeInstaller.processModpack(CurseForgeInstaller.java:507)
		at me.itzg.helpers.curseforge.CurseForgeInstaller.resolveModpackFileAndProcess(CurseForgeInstaller.java:328)
		at me.itzg.helpers.curseforge.CurseForgeInstaller.installByRetrievingModpackZip(CurseForgeInstaller.java:196)
		at me.itzg.helpers.curseforge.CurseForgeInstaller.lambda$install$4(CurseForgeInstaller.java:130)
		at me.itzg.helpers.curseforge.CurseForgeInstaller.install(CurseForgeInstaller.java:165)
		at me.itzg.helpers.curseforge.CurseForgeInstaller.install(CurseForgeInstaller.java:129)
		at me.itzg.helpers.curseforge.InstallCurseForgeCommand.call(InstallCurseForgeCommand.java:202)
		at me.itzg.helpers.curseforge.InstallCurseForgeCommand.call(InstallCurseForgeCommand.java:28)
		at picocli.CommandLine.executeUserObject(CommandLine.java:2041)
		at picocli.CommandLine.access$1500(CommandLine.java:148)
		at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)
		at picocli.CommandLine$RunLast.handle(CommandLine.java:2453)
		at picocli.CommandLine$RunLast.handle(CommandLine.java:2415)
		at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)
		at picocli.CommandLine$RunLast.execute(CommandLine.java:2417)
		at picocli.CommandLine.execute(CommandLine.java:2170)
		at me.itzg.helpers.McImageHelper.main(McImageHelper.java:149)

Caused by: com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `me.itzg.helpers.curseforge.model.ModLoaderType` from String "6": not one of the values accepted for Enum class: [0, 1, 2, 3, 4, 5]
 at [Source: (reactor.netty.ByteBufMono$ReleasingInputStream); line: 1, column: 11029] (through reference chain: me.itzg.helpers.curseforge.model.GetModResponse["data"]->me.itzg.helpers.curseforge.model.CurseForgeMod["latestFilesIndexes"]->java.util.ArrayList[1]->me.itzg.helpers.curseforge.model.FileIndex["modLoader"])
	at com.fasterxml.jackson.databind.exc.InvalidFormatException.from(InvalidFormatException.java:67)
	at com.fasterxml.jackson.databind.DeserializationContext.weirdStringException(DeserializationContext.java:2002)
	at com.fasterxml.jackson.databind.DeserializationContext.handleWeirdStringValue(DeserializationContext.java:1230)
	at com.fasterxml.jackson.databind.deser.std.EnumDeserializer._deserializeAltString(EnumDeserializer.java:415)
	at com.fasterxml.jackson.databind.deser.std.EnumDeserializer._fromString(EnumDeserializer.java:279)
	at com.fasterxml.jackson.databind.deser.std.EnumDeserializer.deserialize(EnumDeserializer.java:258)
	at com.fasterxml.jackson.databind.deser.impl.MethodProperty.deserializeAndSet(MethodProperty.java:129)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:314)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:177)
	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer._deserializeFromArray(CollectionDeserializer.java:359)
	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:244)
	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:28)
	at com.fasterxml.jackson.databind.deser.impl.MethodProperty.deserializeAndSet(MethodProperty.java:129)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:314)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:177)
	at com.fasterxml.jackson.databind.deser.impl.MethodProperty.deserializeAndSet(MethodProperty.java:129)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:314)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:177)
	at com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:323)
	at com.fasterxml.jackson.databind.ObjectReader._bindAndClose(ObjectReader.java:2105)
	at com.fasterxml.jackson.databind.ObjectReader.readValue(ObjectReader.java:1481)
	at me.itzg.helpers.http.ObjectFetchBuilder.lambda$handleResponse$1(ObjectFetchBuilder.java:85)
	... 9 common frames omitted

[init] ERROR failed to auto-install CurseForge 
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -42,7 +42,7 @@ RUN easy-add --var os=${TARGETOS} --var arch=${TARGETARCH}${TARGETVARIANT} \
   --var version=1.9.0 --var app=mc-server-runner --file {{.app}} \
   --from https://github.com/itzg/{{.app}}/releases/download/{{.version}}/{{.app}}_{{.version}}_{{.os}}_{{.arch}}.tar.gz
 
-ARG MC_HELPER_VERSION=1.32.8
+ARG MC_HELPER_VERSION=1.32.9
 ARG MC_HELPER_BASE_URL=https://github.com/itzg/mc-image-helper/releases/download/${MC_HELPER_VERSION}
 # used for cache busting local copy of mc-image-helper
 ARG MC_HELPER_REV=1

